scripts/postinstall.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd /var/praekelt/goddard-captive-portal
+npm install
+
+mkdir -p /var/goddard/apps
+ln -s /var/praekelt/goddard-captive-portal /var/goddard/apps/captiveportal-redesign
```

Comment:
```
bruce@trusty:~$ mkdir -p /var/tmp/foo/tmp/
bruce@trusty:~$ mkdir -p /var/tmp/foo/tmp/
bruce@trusty:~$ ln -s /var/log /var/tmp/foo/tmp/
bruce@trusty:~$ ln -s /var/log /var/tmp/foo/tmp/
ln: failed to create symbolic link '/var/tmp/foo/tmp/log': File exists
bruce@trusty:~$ 
```

Not too sure of the definition of idempotent as it relates to this, but here's the test.